ai-platform-unified/notebooks/ucaip_customjob_image.ipynb

```diff
@@ -0,0 +1,2540 @@
+{
```

Comment:
<ol><li>Can we replace both instances of "Cloud AI Platform" with "AI Platform (Unified)"?</li><li>The AI Platform pricing like should change from https://cloud.google.com/ml-engine/docs/pricing to https://cloud.google.com/ai-platform-unified/pricing</li><li>I don't think the Pricing Calculator supports AI Platform (Unified) currently, so let's remove that section. So the whole sentence can read as follows: "Learn about AI Platform (Unified) pricing and Cloud Storage pricing."</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/240/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>